functional-programming/introduction-to-currying-and-partial-application.js

```diff
@@ -0,0 +1,11 @@
+function add(x) {
+// Only change code below this line
+ return function(y) {
```

Comment:
adjust the alignment inside brackets
the r of the word return must be align with the close bracket }.